### ISSUE TITLE
Panel plugin not found: grafana-groupedbarchart-panel

### ISSUE THREAD

**Issue by bhargavisabbisetty** [gipong/grafana-groupedbarchart-panel#2]
Hi,
When I try to access the following plugin it throws the above error. I am sure that I have placed your file in correct place as image is getting displayed. So, can you help me to solve this problem.

**Comment by gipong** [comment]
This plugin is not support in Grafana v6.0 now.

**Comment by holgerbrandl** [comment]
Any plans for v6+ support? 

I just filed a similar ticket with https://github.com/grafana/grafana/issues/19994 earliers today, but god knows when/if they will pick it up, so having a solid plugin for barchart use-cases in v6 would be awesome.

**Comment by gipong** [comment]
I have downloaded the grafana v6, this plugin can work.
But i will try to create the plugin using React.

**Comment by holgerbrandl** [comment]
Indeed. it works with v6.4. Not sure why, but with the installation instruction from this plugins README it did not work, so I used ` grafana-cli --pluginUrl https://github.com/gipong/grafana-groupedbarchart-panel/archive/master.zip plugins install groupedbarchart` to install it.

**Comment by SaturnusDJ** [comment]
Could get the plugin installed with the above command (thanks!) but no render in Grafana. Tried that same command for https://github.com/CorpGlory/grafana-multibar-graph-panel and surprisingly it worked.

**Comment by jayaddison-collabora** [comment]
Thanks @holgerbrandl; your installation method works for me using a Grafana v6.x instance.

And thank you @gipong for the plugin.  Maybe the README could be updated with installation instructions for Grafana v6 and v7?

**Comment by gipong** [comment]
I update the readme!

**Comment by ghost** [comment]
Thank you @gipong!
